### PR TITLE
Feat/challenge 3 loops conditionals

### PR DESCRIPTION
**Pull request by adrianagrd**

### CODE REVIEW

File: challenge_03_loops-conditionals/03_JavaScript_foundations/.gitignore
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/.gitignore:1]: ⚠️ FIX: move this file to root level

File: challenge_03_loops-conditionals/03_JavaScript_foundations/.prettierignore
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/.prettierignore:1]: ⚠️ FIX: move this file to root level

File: challenge_03_loops-conditionals/03_JavaScript_foundations/package-lock.json
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/package-lock.json:1]: ⚠️ FIX: move this file to root level

File: challenge_03_loops-conditionals/03_JavaScript_foundations/package.json
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/package.json:1]: ⚠️ FIX: move this file to root level

File: challenge_03_loops-conditionals/03_JavaScript_foundations/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/README.md:1]: `README.md` should live at root level. In fact, it is already there (you can check it if you move to project root)

⚠️ FIX: remove this file from this folder

File: challenge_03_loops-conditionals/03_JavaScript_foundations/index.js
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/index.js:1]: ⚠️ FIX: remove file

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_00.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    6. If it is neither "cat" nor "dog", print out "Must be an alien"
+    7. At the end of the program print out: "--- Animal check complete ---"
+    8. Change animal to different values to make sure it works in all cases
+*/
+
+console.log("--- Animal Checker Program ---");
+
+let animal = "cat";
+
+if (animal === "cat") {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/conditionals/exercise_00.js:16]: ⚠️ FIX: Change animal to different values to make sure it works in all cases

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+print("Select an option: ");
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js]: This is the output of your script...
```
ReferenceError: print is not defined
```

...have you tested it before pushing? 🤔

⚠️ FIX: spot the 🐛 and fix it.

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  case "1":
+    evenNumbers();
+    break;
+  case "2":
+    oddNumbers();
+    break;
+}
+
+//even numbers
+for (let i = 10; i <= 40; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_00.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_02.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/loops/exercise_02.js:22]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/*
+    1. Create a while loop that will print out all the even numbers between 10 and 40.
+
+    2. Create a while loop that will print out all the odd numbers between 10 and 40.
+*/
+
+print("Select an option: ");
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js]: This is the output of your script...
```
ReferenceError: print is not defined
```

...have you tested it before pushing? 🤔

⚠️ FIX: spot the 🐛 and fix it

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  case "1":
+    evenNumbers();
+    break;
+  case "2":
+    oddNumbers();
+    break;
+}
+
+//even numbers
+let i = 10;
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_01.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable

File: challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_02.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+let i = 1;
```
Comment (tictools) [inline challenge_03_loops-conditionals/03_JavaScript_foundations/loops-conditionals/while/exercise_02.js:22]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case?

⚠️ FIX: rename properly `i` variable

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by tictools — APPROVED** [review]

**Review by aalvar0 — APPROVED** [review]

**Pull request merged**